README.md

```diff
@@ -171,6 +172,35 @@ Options:
 
 ```
 
+### `lint-messages`
+
```

Comment:
For consistency with other sections of this readme we should include the command line options displayed upon running `mastarm lint-messages --help`.